scripts/projectfileheaderfix.sh

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
```

Comment:
That is an error with the move.  Instead of moving the file from where it was found to where it should be, I was moving the file from where the project file said it would be, to where it should be.  This makes sense for files changed in Xcode, but errors for files manually moved for testing.  I have now changed it to move the file from where the file is found, since this works in both scenarios.